test/service/test_manifest.py

```diff
@@ -340,16 +340,24 @@ def test_manifest_zarr(self):
             with self.subTest(is_single_part=single_part):
                 with mock.patch.object(type(config), 'disable_multipart_manifests', single_part):
                     response = self._get_manifest(ManifestFormat.compact, filters)
-                    self.assertEqual(200, response.status_code, 'Unable to download manifest')
+                    self.assertEqual(200, response.status_code)
                     # Cannot use response.iter_lines() because of https://github.com/psf/requests/issues/3980
                     lines = response.content.decode('utf-8').splitlines()
                     tsv_file = csv.DictReader(lines, delimiter='\t')
                     rows = list(tsv_file)
                     self.assertEqual(len(rows), 13)  # 12 related file, one original
-                    self.assertEqual(len(rows), len({row['file_uuid'] for row in rows}), 'Rows are not unique')
+                    self.assertEqual(len(rows), len({row['file_uuid'] for row in rows}))
                     # 2 pairs of files out of the 13 have matching checksums
-                    self.assertEqual(11, len({row['file_sha256'] for row in rows}), 'Incorrect unique sha256 count')
-                    self.assertEqual(11, len({row['file_crc32c'] for row in rows}), 'Incorrect unique crc32c count')
+                    self.assertEqual(11, len({row['file_sha256'] for row in rows}))
+                    self.assertEqual(11, len({row['file_crc32c'] for row in rows}))
+
+                    response = self._get_manifest(ManifestFormat.curl, filters)
+                    self.assertEqual(200, response.status_code)
+
+                    lines = response.content.decode('utf-8').splitlines()
```

Comment:
utf-8 is the default encoding. Fix all places in this method in separate commit.